tests/SandboxRequire.spec.ts

```diff
@@ -1,3 +1,4 @@
+import * as path from 'path';

```

Comment:
Pretty nitpicky, but I think our convention in other files has been to import our own files first, then any 3rd parties.